packages/strapi/lib/services/__tests__/core-store.test.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+const { createCoreStore } = require('../core-store');
+
+const data = {
+  key: 'testKey',
+  value: 'testValue',
+};
+
+const queryRes = {
+  id: 1,
+  key: 'testKey',
+  value: 'testValue',
+  type: 'string',
+  environment: '',
+  tag: '',
+};
+
+const where = {
+  key: 'plugin_testName_testKey',
+  environment: 'test',
+  tag: '',
+};
+
+describe('Core Store', () => {
+  test.only('Set key in empty store', async () => {
```

Comment:
you are use test.only everywhere instead of test(

Reply:
Oh! my bad forgot to remove that